content/en/guides/features/rendering-modes.md

```diff
@@ -24,18 +24,24 @@ export default {
 ```
 
 <base-alert type="info">
+
 You do not need to add this to your nuxt config in order for universal mode to be applied as the default mode is universal.
+
 </base-alert>
 
 ## SPA
 
-`mode: 'spa'`: No server-side rendering, only client-side rendering.
+<base-alert type="warning">
+
+deprecated `mode: 'spa'` please use `ssr: false`.
```

Comment:
Not consistent with https://github.com/nuxt/nuxtjs.org/pull/675/files#diff-80c9a15d34c22cee073af573d39a115dR11

Reply:
Deprecated: please use `ssr: false` instead of `mode: spa` added like in previous suggestions. Is that what you meant?

Reply:
removed deprecated warning